src/test/java/com/openlattice/shuttle/transforms/TransformTest.java

```diff
@@ -328,20 +328,96 @@ public void testReplaceTransform() {
 
     }
 
-    @Test(timeout=3000)
+    @Test(timeout=4000)
```

Comment:
This is fine for now but in unit tests I'd prefer mocking out the `GeocoderTransform` API so that this doesn't potentially take 3s+ every time you want to run unit tests.

Reply:
+1

Reply:
I'm not sure how to do this without making the test tautological. Could you explain what you mean by mocking out?

Reply:
Set it back to 3s.  Speeding up geocoder is not part of shuttle.